docs/developer/how-to/contribute.rst

```diff
@@ -1 +1 @@
-.. include:: ../../CONTRIBUTING.rst
+.. include:: ../../../.github/CONTRIBUTING.rst
```

Comment:
Please can you also make this change in docs/user/reference/contributing. That's what is causing the docs CI failure